code/modules/power/apc.dm

```diff
@@ -461,17 +461,23 @@ GLOBAL_LIST_INIT(apc_wire_descriptions, list(
 			overlays = 0
 
 		if(!(stat & (BROKEN|MAINT)) && update_state & UPSTATE_ALLGOOD)
-			overlays += emissive_appearance(status_overlays_lock[locked + 1].icon, status_overlays_lock[locked + 1].icon_state)
-			overlays += mutable_appearance(status_overlays_lock[locked + 1].icon, status_overlays_lock[locked + 1].icon_state)
-			overlays += emissive_appearance(status_overlays_charging[charging + 1].icon, status_overlays_charging[charging + 1].icon_state)
-			overlays += mutable_appearance(status_overlays_charging[charging + 1].icon, status_overlays_charging[charging + 1].icon_state)
+			var/image/_lock = status_overlays_lock[locked + 1]
+			var/image/_charging = status_overlays_charging[charging + 1]
+			var/image/_equipment = status_overlays_equipment[equipment + 1]
+			var/image/_lighting = status_overlays_lighting[lighting + 1]
+			var/image/_environ = status_overlays_environ[environ + 1]
```

Comment:
you could do this or you can add a typehint to the list

Reply:
Looks cleaner imo than accessing the same index over and over